src/Controllers/SchemaController.js

```diff
@@ -151,10 +151,16 @@ const defaultColumns: { [string]: SchemaFields } = Object.freeze({
 });
 
 const requiredColumns = Object.freeze({
-  _Product: ['productIdentifier', 'icon', 'order', 'title', 'subtitle'],
-  _Role: ['name', 'ACL'],
+  read: {
```

Comment:
Just one this last nit, could you add a comment above `requiredColumns` and explain shortly what it defines? These centralized definitions are great, but the risk is that someone misunderstands their purpose, changes them and suddenly there may be vulnerability or bug in a totally unrelated pat of the code.